src/Explorer/SplashScreen/SplashScreen.tsx

```diff
@@ -209,20 +307,27 @@ export class SplashScreenComponentAdapter implements ReactAdapter {
         iconSrc: null,
         title: "Data Modeling",
         description: "Learn more about modeling",
-        onClick: () => window.open(SplashScreenComponentAdapter.dataModelingUrl),
+        onClick: () => window.open(SplashScreen.dataModelingUrl),
       },
       {
         iconSrc: null,
         title: "Cost & Throughput Calculation",
         description: "Learn more about cost calculation",
-        onClick: () => window.open(SplashScreenComponentAdapter.throughputEstimatorUrl),
+        onClick: () => window.open(SplashScreen.throughputEstimatorUrl),
       },
       {
         iconSrc: null,
         title: "Configure automatic failover",
         description: "Learn more about Cosmos DB high-availability",
-        onClick: () => window.open(SplashScreenComponentAdapter.failoverUrl),
+        onClick: () => window.open(SplashScreen.failoverUrl),
       },
     ];
   }
+
+  private onSplashScreenItemKeyPress(event: React.KeyboardEvent, callback: () => void) {
```

Comment:
Note that we now have an `<AccessibleElement as="li" />` which does this for you.

Reply:
Fantastic, good to know. I've seen this pattern in a few different places, so this seems like a good PR to do in one fell swoop?